src/t8_element_c_interface.cxx

```diff
@@ -116,8 +116,7 @@ t8_element_sibling (t8_eclass_scheme_c *ts,
 }
 
 int
-t8_element_num_corners (const t8_eclass_scheme_c *ts,
-                        const t8_element_t *elem)
+t8_element_num_corners (t8_eclass_scheme_c *ts, const t8_element_t *elem)
```

Comment:
I also believe #377 was referring to the `const` at the end of the declaration as in `int foo() const`.

Reply:
Sure, but the former differed from the declaration in `t8_element_c_interface.h`. Maybe, this is why the compiler did not include the symbol. Removing the `const` for `ts` is in accordance to the convention of the other `t8_element_*` routines. If you want to change that, than this should be changed for all routines in a separate PR.

Reply:
`Sure, but the former differed from the declaration in t8_element_c_interface.h`
ahhh, so then we should add the `const` to the C interface instead of removing it from the `C++` one?

Reply:
Okay, it starts to clear up now for me...did not completely understand the issue so far.

Can we say:

If the C++ interface hast the const modifier for the function, than the C interface needs the `const t8_eclass_scheme_c`.
And will this then work with the Julia interface?


I understand that most functions are missing the const modifier and the `const t8_eclass_scheme_c`.
However, going forward this should change towards using const everywhere where a function does not modify the class.
Not in the direction of using it less.

Or do i still understand it wrong and the solution that works is:

Have the const modifier for the function in the C++ interface, but drop it for `eclass_scheme_c` in the C interface?

Reply:
To add to the discussion `t8_element_num_faces` works/produces the right symbol. The differences are the `const eclass_scheme_c`
https://github.com/DLR-AMR/t8code/blob/main/src/t8_element_c_interface.cxx#L128
and
https://github.com/DLR-AMR/t8code/blob/main/src/t8_element_cxx.hxx#L169 the method declaration is non-const compared to
https://github.com/DLR-AMR/t8code/blob/main/src/t8_element_cxx.hxx#L162

I'm also not sure what the right combination should be.